### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by litvintech**

### CODE REVIEW

**Comment by litvintech** [comment]
> In my opinion, the go-cyber readme should contain the following structure:
> 
> * a clear vision (statement)
> * describe the work of cyber as a computer (like we do in the WP, in terms of distributed tech)
> * compare it to google (image)
> * explain how the protocol works (we can use the version from the WP) in short steps
> * Validator section (one paragraph with links)
> * Gitcoin (image)
> * license (one sentence)
> * resources  (5-6 links)
> 
> Just my 2 cents

All this resolved

File: README.md [hunk 1]
```diff
@@ -26,251 +44,147 @@
...
-```
-
-## Inter Knowledge Protocol for Relevance Machines (over IBC)
-```
-Protocol prototyping on:
-1. Subgraph Interchange
-2. CIDs Interchange
-3. Ranks Interchange
-```
+Explore our [network](https://cyber.page/brain) and participate in [Game of Links](https://cybercongress.ai/game-of-links/)
```
Comment (xhipster) [inline README.md:133]: Я бы слал напрямую на страницу с мясом - https://cyber.page/gol

File: README.md [hunk 2]
```diff
@@ -26,251 +44,147 @@
...
+Explore the documentation in our [knowledge base](https://cybercongress.ai/docs/cyberd/cyberd/).
+
+- Guides:
+  - [ELI-5](https://github.com/cybercongress/congress/blob/master/ecosystem/ELI-5%20FAQ.md)
+  - [Cyber Homestead](https://github.com/cybercongress/congress/blob/master/ecosystem/Cyber%20Homestead%20doc.md)
+  - [cyber~Ecosystem development](https://github.com/cybercongress/congress/blob/master/ecosystem/cyber~Ecosystem%20development%20paper.md)
+  - [Usecases](https://github.com/cybercongress/congress/blob/master/ecosystem/usecases.md)
+- Usage:
+  - [CLI guide](./docs/cyberd/ultimate-commands-guide.md")
+  - [Get EUL](./docs/cyberd/get_CYB.md")
```
Comment (xhipster) [inline README.md:145]: У меня есть проект страницы - ./docs/cyberd/get_CYB.md идет вот сюда https://cyber.page/get вставь эту ссылку

Comment (serejandmyself) [reply README.md:145]: Speak English blya =)

**Pull request merged**